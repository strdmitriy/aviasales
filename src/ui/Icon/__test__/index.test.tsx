import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Icon } from "../";


describe('Testing Icon/Logo', () => {
    test("Render Icon", () => {
        const renderer = render(
            <Icon image="DP@2x"/>
        );
        expect(renderer).toMatchSnapshot();
    });
});