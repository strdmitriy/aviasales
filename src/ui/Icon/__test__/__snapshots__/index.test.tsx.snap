// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Icon/Logo Render Icon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <img
        class="Icon__Image-sc-11x6y0b-0 fmeCBF"
        height="36"
        src="//pics.avs.io/99/36/DP@2x@2x.png"
        width="110"
      />
    </div>
  </body>,
  "container": <div>
    <img
      class="Icon__Image-sc-11x6y0b-0 fmeCBF"
      height="36"
      src="//pics.avs.io/99/36/DP@2x@2x.png"
      width="110"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
